src/input.cpp

```diff
@@ -1355,6 +1356,34 @@ cata::optional<tripoint> input_context::get_coordinates( const catacurses::windo
 }
 #endif
 
+std::tuple<point, bool> input_context::get_coordinates_inventory( const catacurses::window
```

Comment:
Why does the method name contain "inventory"? I see nothing inventory-specific in this method.

Reply:
Ah, that's quite true, but currently it is for inventory only, maybe in future can change its name if it is going to be used by other thing.

Reply:
I renamed the method with get_coordinates_text